docs/pages/setup/reference/terraform-provider.mdx

```diff
@@ -167,6 +167,9 @@ Metadata specifies a resource name, description, labels and optional expiry date
 |              `namespace`|     string |          | Resource namespace ("default" by default) |
 |                `expires`|     string |          | Expiry time in RFC8829 format             |
 
+**version**
+
+`version` is an optional string that specifies the role version (default `v3`). 
```

Comment:
isn't default a v4 now?

Reply:
@stevenGravy may be make it a variable instead of hardcode?

Reply:
In Teleport 9 the default is `v5` I believe. @xacrimon Can you confirm?

Reply:
Yup

Reply:
so this is still accurate in that it's the terraform, not a sample

Reply:
@klizhentas I think you could make it variable.  this was the reference so just wanted it documented that we have this as a string